YourToDo/templates/planner/planner.html

```diff
@@ -8,180 +8,171 @@
         <!-- Table containing top content (could potentially be used for all views) -->
         <table class="planner-container">
             <!-- Column Specification: Left sidebar, 7 days -->
-            <col class="planner-week-left-sidebar">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-right-sidebar">
 
             <!-- Top Row: Mini Calendar/Title -->
             <tr>
-                <!-- Mini Calendar -->
-                <td rowspan="3">
-                    <script type="text/javascript">
-                        var miniCal = new miniCal();
-                        miniCal.generateHTML();
-                        document.write(miniCal.getHTML());
-                    </script>
-                </td>
                 <!-- Planner Title #TODO Let users name their own titles -->
-                <td class="planner-title" colspan="7">
+                <td class="planner-title" colspan="8">
                     My Planner
                     <!-- {{ user.planner.title }} (possibly) -->
                 </td>
             </tr><!-- End Top Row -->
 
             <!-- Top content -->
-            <tr class="planner-week-top-content">
-                <td class="planner-date-selectors" colspan="4">
-                    <span class="glyphicon glyphicon-chevron-left"></span>
-                    {{ date | date:"F" }}
-                    <span class="glyphicon glyphicon-chevron-right"></span>
-
-                    <!-- Weekly dates (i.e. "11/15 - 11/22") -->
+            <tr class="planner-month-top-content">
+                <td class="planner-month-date-selectors" colspan="7">
                     <span class="glyphicon glyphicon-chevron-left"></span>
-                    <!-- #TODO get date range for the week (sunday-saturday) -->
                     {{ date | date:"F" }}
                     <span class="glyphicon glyphicon-chevron-right"></span>
                 </td>
-                <td class="planner-layout-selectors" colspan="3">
+                <td class="planner-month-layout-selectors" colspan="1">
```

Comment:
There's going to be several merge conflicts with the top portion because of work that I've done to these, but I'll handle that.